src/uwtools/drivers/ungrib.py

```diff
@@ -0,0 +1,175 @@
+"""
+A driver for the ungrib component.
+"""
+
+import os
+import stat
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict
+
+from iotaa import asset, dryrun, task, tasks
+
+from uwtools.config.formats.nml import NMLConfig
+from uwtools.drivers.driver import Driver
+from uwtools.strings import STR
+from uwtools.utils.tasks import file
+
+
+class Ungrib(Driver):
+    """
+    A driver for ungrib.
+    """
+
+    _driver_name = STR.ungrib
```

Comment:
This ended up as a private method with a `@property` decorator in a recent omnibus PR, IIR.